compiler/rustc_codegen_rmc/src/codegen/rvalue.rs

```diff
@@ -760,37 +779,37 @@ impl<'tcx> GotocCtx<'tcx> {
             // for it. Build and insert a function that just calls an unimplemented block
             // to maintain soundness.
             let drop_sym_name = format!("{}_unimplemented", self.symbol_name(drop_instance));
-
-            // Function body
-            let unimplemented = self
-                .codegen_unimplemented(
-                    format!("drop_in_place for {}", drop_sym_name).as_str(),
-                    Type::empty(),
+            let drop_sym = self.ensure(&drop_sym_name, |ctx, name| {
+                // Function body
+                let unimplemented = ctx
+                    .codegen_unimplemented(
+                        format!("drop_in_place for {}", drop_sym_name).as_str(),
+                        Type::empty(),
+                        Location::none(),
+                        "https://github.com/model-checking/rmc/issues/281",
+                    )
+                    .as_stmt(Location::none());
+
+                // Declare symbol for the single, self parameter
```

Comment:
Not for this PR, but this feels like something that would happen often enough to be worth a helper function